samples/cpp/tutorial_code/calib3d/camera_calibration/camera_calibration.cpp

```diff
@@ -567,25 +594,32 @@ static void saveCameraParams( Settings& s, Size& imageSize, Mat& cameraMatrix, M
 
     if (s.flag)
     {
+        membuf sbuf(buf, buf + sizeof(buf));
+        std::ostream output(&sbuf);
```

Comment:
I believe [std::stringstream](http://www.cplusplus.com/reference/sstream/stringstream/) is more [reliable](http://www.cplusplus.com/reference/sstream/stringstream/stringstream/) here.